static/source/guides/getting_started.html.slim

```diff
@@ -48,54 +48,86 @@ section.guide
       <code>
 
-      ### Creating a GitHub account for Danger to use
+      ### Creating a bot account for Danger to use
 
       This is optional. Pragmatically, you want to do this though.
 
-      In order to get the most out of Danger, we recommend giving her the ability to post comments in your code review. This is a regular GitHub account, but depending on whether you are working on a private or public project, you will want to give different levels of access to this bot. You are allowed to have [one bot per GitHub account][github_bots].
+      [bundler]: http://bundler.io
+      [cp_bundler]: https://guides.cocoapods.org/using/a-gemfile.html
+      [without_bundler]: #installation-without-bundler
+      [vs_code]: https://marketplace.visualstudio.com/items?itemName=Orta.vscode-danger
+
+    div.request_sources.selectors
+      ul
+        li.github.highlighted GitHub
+        li.gitlab GitLab
+
+      div.github
+        markdown:
+
+          In order to get the most out of Danger, we recommend giving her the ability to post comments in your code review. This is a regular GitHub account, but depending on whether you are working on a private or public project, you will want to give different levels of access to this bot. You are allowed to have [one bot per GitHub account][github_bots].
+
+          To get started, open [https://github.com](https://github.com) in a private browser session.
+
+          ##### OSS Projects
+
+          Do not add the bot to your repo or to your organization.
+
+          ##### Closed Source Projects
+
+          Add the bot to your repo or to your organization. The bot requires permission level "Write" to be able to set a PR's status. Note that you _should not_ re-use this bot for OSS projects.
 
-      To get started, open [https://github.com](https://github.com) in a private browser session.
+          ### Setting up an Access Token
 
-      ##### OSS Projects
+          [Here's the link][github_token], you should open this in the private session where you just created the new GitHub account. Again, the rights that you give to the token depend on the openness of your projects.
 
-      Do not add the bot to your repo or to your organization.
+          ##### Tokens for OSS Projects
 
-      ##### Closed Source Projects
+          We recommend giving the token the smallest scope possible. This means just `public_repo`, this scopes limits Danger's abilities to just writing comments on OSS projects. Because the token can be quite easily be extracted from the CI environment, this minimizes the chance for bad actors to cause chaos with it.
 
-      Add the bot to repo or to your organization. The bot requires permission level "Write" to be able to set a PR's status. Note that you _should not_ re-use this bot for OSS projects.
+          ##### Tokens for Closed Source Projects
 
-      ### Setting up an Access Token
+          We recommend giving access to the whole `repo` scope, and its children.
```

Comment:
> We recommend giving the bot access to your project with the "Guest" permission level

Unfortunately commit status is in the developer permission group which is a bit too broad. I guess it could be optional for private repos